### DISCUSSION TITLE
Github Status Bugs

### DISCUSSION THREAD

**Discussion by cmdNiels** [community/community#16590] [Other Feature Feedback, Questions, & Ideas]
Hello, when I tried to set my GitHub profile status I came across some bugs.

There is a invalid Emoji on windows 10 (smiling_face_with_tear).
![Github Invalid Emoji](https://user-images.githubusercontent.com/73907007/168430205-f3964a88-da89-4d04-ab75-49161d8f9d3b.png)

There also were double emoji's (I don't know if this is intended).
![Github Double Emoji](https://user-images.githubusercontent.com/73907007/168430238-a9089901-d46a-42a8-aa6d-538728512716.png)

I thought I should report this. Could someone let me know if this is intended?

**Comment by ponjjar** [comment]
This is a new emoji (v13) . They need to update fonts.

**Comment by rilysh** [comment]
Not a problem in my side. I assume you need to update Windows 10 with latest version of Segoe UI emoji font. Personally, I'm not a Windows user but if I remember correctly, emojis that Windows uses, are part of Segoe UI font.

Which build version of Windows 10 you're using?

**Comment by d-coder111** [comment]
> Hello, when I tried to set my GitHub profile status I came across some bugs.
> 
> There is a invalid Emoji on windows 10 (smiling_face_with_tear). ![Github Invalid Emoji](https://user-images.githubusercontent.com/73907007/168430205-f3964a88-da89-4d04-ab75-49161d8f9d3b.png)
> 
> There also were double emoji's (I don't know if this is intended). ![Github Double Emoji](https://user-images.githubusercontent.com/73907007/168430238-a9089901-d46a-42a8-aa6d-538728512716.png)
> 
> I thought I should report this. Could someone let me know if this is intended?

I believe that there's no issue regarding the same as everything is alright on my end.

**Comment by melvincwng** [comment]
Hi @cmdNiels, you can report bugs at this link: https://support.github.com/contact/bug-report

Hope this clarifies

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
